Python/bytecodes.c

```diff
@@ -1204,14 +1215,16 @@ dummy_func(
 
             PyObject *retval_o;
             assert(frame->owner != FRAME_OWNED_BY_INTERPRETER);
+            _PyStackRef tmp = PyStackRef_MakeHeapSafe(v);
+            DEAD(v);
```

Comment:
Can we do something like https://gist.github.com/colesbury/ce98264761c831d42d185f758395933e instead?

Having a `_PyStackRef` that only exists on the C stack worries me especially with the `PyObject_CallMethodOneArg()` call below.

Reply:
Yeah, good call

Reply:
Fixed, you should be able to write
```
    _PyStackRef tmp = PyStackRef_MakeHeapSafe(v);
    DEAD(v);
    v = tmp;
```
now.